core/amber/build.sbt

```diff
@@ -125,16 +125,6 @@ val googleServiceDependencies = Seq(
   "com.sun.mail" % "javax.mail" % "1.6.2"
 )
 
-/////////////////////////////////////////////////////////////////////////////
-// Arrow related
-val arrowVersion = "14.0.1"
-val arrowDependencies = Seq(
-  // https://mvnrepository.com/artifact/org.apache.arrow/flight-grpc
-  "org.apache.arrow" % "flight-grpc" % arrowVersion,
-  // https://mvnrepository.com/artifact/org.apache.arrow/flight-core
-  "org.apache.arrow" % "flight-core" % arrowVersion
-)
-
```

Comment:
please test if Python UDF still works. as you moved the dependencies.

Reply:
Python UDF works well.